### DISCUSSION TITLE
Renovate won't update .nvmrc to a major version

### DISCUSSION THREAD

**Discussion by Wassim-AB** [renovatebot/renovate#18453] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.241

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### If you're self-hosting Renovate, tell us what version of the platform you run.

GitLab Enterprise Edition 15.4.2-ee

### Was this something which used to work for you, and then stopped?

I never saw this working

### Describe the bug

Hello,

I'm currently using Renovate as a CronJob in a Kubernetes Cluster to manage my Node dependencies. So far in my test repository I only have : 

- A .gitlab-ci.yml file that uses a Node image
- A Dockerfile that uses a Node image
- A initialized package.json
- A .nvmrc file
- A renovate.json file that points to my Renovate configuration repository.

All of these files are using the v16.15.0 of Node.

In my Renovate configuration repository called "renovate-config-wass" I only have this rule that enables Node major updated :

```json
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",

    "description": "Group major updates of Node and its types.",
    "separateMultipleMajor": true,
    "packageRules": [
      {
        "groupName": "Node",
        "matchPackageNames": ["@types/node", "node", "nodejs/node"],
        "matchManagers": ["nvm"],
        "matchUpdateTypes": ["major"]
      }
    ]
}
```

I also have, in this same repository, a default.json file that extends this rule :

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "description": "Default presets",
  "extends": [
    "local>xxx/xx/renovate-config-wass:nodeMajors"
  ]
}
```

Finally in my test repository, I have a renovate.json file that points to my Renovate configuration repository :

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["local>xxx/xx/renovate-config-wass"]
}
```

When running the CronJob, Renovate can successfully find my configuration repository and create 4 PRs (16.15.0 -> 16.18.0, 16->17, 16->18, 16->19), grouping every file mentioned above in each PR. However, Renovate doesn't include the .nvmrc in the major updates, it only detects an upgrade for the 16.15.0 to 16.18.0 PR although I mentioned in my configuration file that I allow major updated for the nvm manager. Am I missing something in my configuration ?

You'll find here two public repositories that are copies of my configurations : 
https://github.com/Wassim-AB?tab=repositories

I've also copied logs in the "Relevant debug logs" section.

Thank you for your time.

### Relevant debug logs

<details><summary>Logs</summary>

```
"nvm": [
           {
             "packageFile": ".nvmrc",
             "deps": [
               {
                 "depName": "node",
                 "currentValue": "16.18.0",
                 "datasource": "github-tags",
                 "packageName": "nodejs/node",
                 "depIndex": 0,
                 "updates": [],
                 "warnings": [],
                 "versioning": "node",
                 "sourceUrl": "https://github.com/nodejs/node",
                 "currentVersion": "16.18.0",
                 "fixedVersion": "16.18.0"
               }
             ]
           }
         ]
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by viceice** [comment, marked as answer]
nvm manager uses node versioning by default, which only allows lts and latest node lts is still v16. v18 will be lts next week or so.

https://github.com/renovatebot/renovate/blob/da244e3b9c7010805c9fcedc4e3d4d204b14cd14/lib/modules/manager/nvm/index.ts#L14

**Reply by Wassim-AB** [reply]
Ok i get it, thank you for your answer !
